### PR TITLE
[#9667] Standardize look on feedback session details card

### LINKED ISSUE #9667: Standardise look of feedback session details card
- **Environment**:  `master` branch at commit a1c21185

**Steps to reproduce**
1. Go to a feedback session submission page (`web/student/sessions/submission?courseid=test.exa-demo&fsname=Session%20with%20different%20question%20types`)
2. Go to a feedback session results page (`web/student/sessions/result?courseid=test.exa-demo&fsname=Session%20with%20different%20question%20types`)

**Expected behaviour**
The card describing the session should look the same on both pages

**Actual behaviour**
The descriptions (course id, session, opening time ...) do not have the same styling

Submit page:
2019-04-12_10h49_21

Results page:
2019-04-12_10h49_16

### PR DESCRIPTION
**Pull request by hppranaav**
Standardized look of the feedback submission page and the feedback result page

Fixes #9667

### CODE REVIEW

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 1]
```diff
@@ -1,19 +1,35 @@
 <h1>
   Feedback Session Results
 </h1>
-<div class="card bg-light top-padded" *ngIf="session">
-  <div class="card-body row" style="padding: 40px;">
-    <label class="col-sm-2 control-label">Course ID:</label>
-    <div class="col-sm-10">{{ session.courseId }}</div>
+<div class="card card-plain" ngtif="session">
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: Should be `*ngIf`. Please spend some time to read up on this [concept](https://angular.io/api/common/NgIf) :)

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+      <div class="col-5">{{ session.feedbackSessionName }}</div>
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Opening Time:</label>
-    <div class="col-sm-10">{{ formattedSessionOpeningTime }}</div>
+    <br/>
+    <div class="row">
+      <div class="col-2 text-right"><b>Opening Time:</b></div>
+      <div class="col-5">{{ session.formatteSessionOpeningTime }}</div>
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: Typo here. Also, `formattedSessionOpeningTime` will do.

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Closing Time:</label>
-    <div class="col-sm-10">{{ formattedSessionClosingTime }}</div>
+    <br/>
+    <div class="row">
+      <div class="col-2 text-right"><b>Closing Time:</b></div>
+      <div class="col-5">{{ session.formattedSessionClosingTime }}</div>
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: `formattedSessionClosingTime` will do.

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 4]
```diff
@@ -1,19 +1,35 @@
...
+    <br/>
+    <div class="row">
+      <div class="col-2 text-right"><b>Session:</b></div>
+      <div class="col-5">{{ session.feedbackSessionName }}</div>
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Opening Time:</label>
-    <div class="col-sm-10">{{ formattedSessionOpeningTime }}</div>
+    <br/>
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: Same here.

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
+    <div class="row">
+      <div class="col-2 text-right"><b>Opening Time:</b></div>
+      <div class="col-5">{{ session.formatteSessionOpeningTime }}</div>
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Closing Time:</label>
-    <div class="col-sm-10">{{ formattedSessionClosingTime }}</div>
+    <br/>
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: Same here.

File: src/web/app/pages-session/session-result-page/session-result-page.component.html [hunk 6]
```diff
@@ -1,19 +1,35 @@
...
+    <div class="row">
+      <div class="col-2 text-right"><b>Course ID:</b></div>
+      <div class="col-5">{{ session.courseId }}</div>
+    </div>
+    <br/>
 
-    <label class="col-sm-2 control-label">Session:</label>
-    <div class="col-sm-10">{{ session.feedbackSessionName }}</div>
+    <br/>
```
Comment (tanhengyeow) [inline src/web/app/pages-session/session-result-page/session-result-page.component.html]: The extra `</br>` here is causing the lines to have unnecessary spacing than what is usually needed:
image

**Comment by hppranaav** [comment]
Thanks @tanhengyeow for reviewing my request. I'll read up a bit more on HTML concepts and apply the fixes wherever needed.

**Comment by hppranaav** [comment]
Hi @tanhengyeow I don't know why the changes t the .scss file are not passing as i feel the syntax and linting is fine.

**Comment by hppranaav** [comment]
Hi @tanhengyeow, I'm not able to understand why the commit to the session-result-page.component.scss file is not passing. I feel the syntax and linting is fine.

**Comment by wkurniawan07** [comment]
> I'm not able to understand why the commit to the `session-result-page.component.scss` file is not passing

@hppranaav your latest commit is fine. What seems to be the problem?

**Comment by xpdavid** [comment]
Close due to inactivity.

**Pull request closed without merging**